docs/references/expo/use-sso.mdx

```diff
@@ -133,44 +131,47 @@ It accepts the following parameters (`StartSSOFlowParams`):
   </Tab>
 
   <Tab>
-    The following example demonstrates how to create a custom SSO sign-in flow with [SAML](/authentication/enterprise-connections/overview#saml).
+    The following example demonstrates how to create a custom SSO sign-in flow with [SAML](docs/authentication/enterprise-connections/overview#saml).
 
     ```tsx {{ filename: 'app/(auth)/sign-in.tsx', collapsible: true }}
-    import React from 'react'
+    import React, { useCallback, useEffect } from 'react'
     import * as WebBrowser from 'expo-web-browser'
-    import { Text, View, Button } from 'react-native'
-    import { Link } from 'expo-router'
-    import { useSSO } from '@clerk/clerk-expo'
     import * as Linking from 'expo-linking'
+    import { useSSO } from '@clerk/clerk-expo'
+    import { View, Button } from 'react-native'
 
     export const useWarmUpBrowser = () => {
-      React.useEffect(() => {
-        // Warm up the android browser to improve UX
-        // https://docs.expo.dev/guides/authentication/#improving-user-experience
+      useEffect(() => {
+        // Preloads the browser for Android devices to reduce authentication load time
+        // See: https://docs.expo.dev/guides/authentication/#improving-user-experience
         void WebBrowser.warmUpAsync()
         return () => {
+          // Cleanup: closes browser when component unmounts
           void WebBrowser.coolDownAsync()
         }
       }, [])
     }
 
+    // Handle any pending authentication sessions
     WebBrowser.maybeCompleteAuthSession()
 
     export default function Page() {
       useWarmUpBrowser()
 
-      const { useSSOFlow } = useSSO({ strategy: 'enterprise_sso' })
+      const { startSSOFlow } = useSSO({ strategy: 'enterprise_sso' })
 
-      const onPress = React.useCallback(async () => {
+      const onPress = useCallback(async () => {
         try {
-          const { createdSessionId, signIn, signUp, setActive } = await useSSOFlow({
+          const { createdSessionId, setActive } = await startSSOFlow({
+            // Specify redirect URL after successful authentication
+            // Must match the scheme defined in app.json
             redirectUrl: Linking.createURL('/dashboard', { scheme: 'myapp' }),
             // User identifier with a email domain that matches a enterprise connection
-            identifier: 'foo@corp.com',
+            identifier: 'email',
```

Comment:
Is `identifier` an email string or a type string (e.g., `email`, `phone`)?

Reply:
Oh, I added this as a variable. This would usually be placed via UI state from an input. 

For the snippet, it's fine to leave it as a hardcoded value.